config/locales/en/en.yml

```diff
@@ -136,6 +136,7 @@ en:
         old_start_date: "Original Start Date"
         scheduled_duration_days: "Scheduled Duration"
         scheduled_start_date: "Scheduled Start Date"
+        scheduled_end_date: "Scheduled End Date"
```

Comment:
Note this was missing. An easy way to catch these is if the label in the form looks like 'Scheduled end date' instead of 'Scheduled End Date' (i.e. not title case).